tests/fixtures/schema/0.5.8/story-fixture-tiny-house.json

```diff
@@ -141,41 +141,35 @@
     "has_published_edition": true,
     "editions": {
       "default": {
-        "edition": "default",
+        "edition_name": "default",
         "edition_date": "2017-04-15T15:00:00Z",
-        "first_published_date": "2017-04-15T12:00:00Z",
-        "publish_date": "2017-04-15T12:00:00Z",
-        "display_date": "2017-04-15T15:00:00Z",
-        "revision_id": "CAIP33ODLZD5LESQ3CH6MLP6IM",
-        "published": true
+        "edition_first_publish_date": "2017-04-15T12:00:00Z",
+        "edition_publish_date": "2017-04-15T12:00:00Z",
+        "edition_display_date": "2017-04-15T15:00:00Z",
+        "edition_revision_id": "CAIP33ODLZD5LESQ3CH6MLP6IM",
+        "edition_publish_status": true
       },
       "print": {
-        "edition": "print",
+        "edition_name": "print",
         "edition_date": "2017-04-16T15:00:00Z",
-        "first_publish_date": "2017-04-16T15:00:00Z",
-        "publish_date": "2017-04-16T15:00:00Z",
-        "display_date": "2017-04-16T15:00:00Z",
-        "revision_id": "CAIP33ODLZD5LESQ3CH6MLP6IM",
-        "published": true
+        "edition_first_publish_date": "2017-04-16T15:00:00Z",
+        "edition_publish_date": "2017-04-16T15:00:00Z",
+        "edition_display_date": "2017-04-16T15:00:00Z",
+        "edition_revision_id": "CAIP33ODLZD5LESQ3CH6MLP6IM",
+        "edition_publish_status": true
       }
     },
     "scheduled_operations": {
       "publish_edition": [{
-        "type": "story_operation",
-        "story_id": "f8f706f0-0acc-11e5-9e39-0db921c47b93",
         "operation": "publish_edition",
-        "date": "2017-04-20T15:00:00Z",
-        "publish_date": "2017-04-15T15:00:00Z",
```

Comment:
It looks like a few fields have gone away here (`publish_date`, `display_date`) - do we need them for anything?  I think you said WebSked would only ever be concerned with `operation_date`, but just want to be sure.